samples/attributes/proposed.json

```diff
@@ -0,0 +1,65 @@
+{
+  "usx": {
+    "version": "3.0",
+    "children": [
+      {
+        "book": {
+          "code": "GEN",
+          "style": "id"
+        }
+      },
+      {
+        "chapter": {
+          "number": "1",
+          "style": "c",
+          "sid": "GEN 1"
+        }
+      },
+      {
+        "para": {
+          "style": "p",
+          "children": [
+            {
+              "verse": {
+                "number": "1",
+                "style": "v",
+                "sid": "GEN 1:1"
+              }
+            },
+            "the first verse ",
+            {
+              "verse": {
+                "eid": "GEN 1:1"
```

Comment:
Dropping eid means the dropping the verse object itself, right? So we use the verse milestone to mark only the beginning not the end. 

Since we are not going to use wrappers or such json object, here is a scenario I can think of where having a verse end milestone might be of use.
```
para1 started.... 
    verse 1 started
    text...
    a character marker
    ..text
    verse 2 started
para1 ended
```
1. when we encounter a character marker(or a notes maker) in between the text of verse 1, for a non-USFM-familiar user it may not be clear for him whether the new object means the end of the verse 1 or not. If he can see the use of an explicit end milestone, that can be easily understood.
2. When we encounter the end of para1 object we are not sure if the verse 2 has ended or is it continuing in the next paragraph object. But if we have an end milestone before para1 end, then there is clarity that verse 2 has indeed ended.

This is not something unavoidable, but just sharing a possible use of having it, since we are keeping verses and chapters as milestones.

Reply:
I am saying this because, I am hoping with a simpler data structure as JSON, can we not expect the user to "know USFM" and have the knowledge of what all can be the end of a verse, but work with just the JSON he has in his hands.

Reply:
@kavitharaju It's true that removing the eids requires some domain-specific knowledge on the part of the processor. I'd argue that this is "How Bibles Work" knowledge rather than USFM knowledge. Also, I believe that the new use case is USFM lexing/tokenisation, rather than direct use in applications. If that's true, I would hope that anyone rolling their own USFM processor using our JSON would understand the basics of how chapters and verses work.

On the other hand, if the use case is as described above, we're not trying to edit this format, which was my main concern with eids, so I guess we could leave them in.